src/components/Areas/Areas.js

```diff
@@ -0,0 +1,19 @@
+function Areas({ parts, side, points, handleChange }) {
```

Comment:
Ja bym już dawała 'export' przy definiowaniu funkcji, czyli na początku:
```suggestion
export function Areas({ parts, side, points, handleChange }) {
```

już bez default'owego exportowania